README.md

```diff
@@ -120,5 +135,32 @@ Here are some the resources you can check:
 
 #### Example
 ```bash
-pleco start --level debug -i 240 -a eu-west-3 -e -r -m -c -l -b -p -s -w -n -u -z -o -y
+pleco start aws --level debug -i 240 -a eu-west-3 -e -r -m -c -l -b -p -s -w -n -u -z -o -y
+```
+
+### Scaleway options
+#### Region selector
+When pleco's look for expired resources, it will do it by scaleway region.
+
+You can set region(s) with:
+```bash
+--scw-regions, -a <region(s)>
+```
+
+For exemple:
+```bash
+-a fr_par
+```
```

Comment:
you can add a CF section: https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/guides/regions_and_zones